### PR TITLE
Added queries to test folder

### PR DESCRIPTION
**Pull request by nikk15**
Added several more queries that are known to work to the query
test folder. More to follow.

### CODE REVIEW

**Comment by nikk15** [comment]
I have added more queries to the test folder. I wasn't able to yesterday as I kept getting HTTP 504 errors but this seems to have been resolved for most queries. 

However, the below errors arose for the below queries when running in the command line:

**jsone.shared.InterpreterError:**
- intermittent_jobs
- intermittent_test_rate
- intermittent_tests
- meta_columns
- task_durations
- test_durations
- tests_config_times
- tests_in_duration

**yaml.parser.ParserError:**
- raw_coverage_nosubdir

**adr.errors.MissingDataError: ActiveData didn't return any data.**
- code_coverage
- raw_coverage

Has this occurred for anyone else? I can investigate this further next week!

File: test/query_tests/try_commit_messages.test [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      - 
+    - - dwillcoxon@mozilla.com
+      - 
+    - - pbone@mozilla.com
+      - 
+    - - gwatson@mozilla.com
+      - 
+    - - ktomlinson@mozilla.com
+      - 
+    - - dhouse@mozilla.com
```
Comment (ahal) [inline test/query_tests/try_commit_messages.test]: Could you invent some fake e-mail addresses for this query? You can use `example.com` as the domain name.

File: test/query_tests/code_coverage_by_suite.test [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    - run.suite.fullname
+    - file
+    - source.file.name
+  meta:
+    content_type: application/json
+    format: table
+  data:
+    - - 
+    - 
+    - 0
```
Comment (ahal) [inline test/query_tests/code_coverage_by_suite.test]: Is this the query that isn't working? Let's just leave broken queries out of it for now.

File: test/query_tests/user_tasks.test [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+expected:
+  header:
+    - repo.push.user
+    - count
+  meta:
+    saved_as: rhwaxTxR
+    content_type: application/json
+    format: table
+  data:
+    - - aciure@mozilla.com
```
Comment (ahal) [inline test/query_tests/user_tasks.test]: Same here, please invent a fake e-mail address

File: test/query_tests/code_coverage_by_suite.test (hunk 2 above)
Comment (nikk15) [reply test/query_tests/code_coverage_by_suite.test]: Actually, this one runs, but doesn't return "data". It returns all the other keys though. The ones I listed above don't run at all - they each have the errors listed. Does this happen to you too?

File: test/query_tests/try_commit_messages.test (hunk 1 above)
Comment (nikk15) [reply test/query_tests/try_commit_messages.test]: Oh gosh! Didn't think of that. Will get right on it. Cheers.

**Pull request merged**